Content.Server/Atmos/EntitySystems/AtmosphereSystem.Monstermos.cs

```diff
@@ -452,7 +454,7 @@ private void ExplosivelyDepressurize(MapGridComponent mapGrid, GridAtmosphereCom
                 AddActiveTile(gridAtmosphere, otherTile);
                 var otherTile2 = otherTile.AdjacentTiles[otherTile.MonstermosInfo.CurrentTransferDirection.ToIndex()];
                 if (otherTile2?.Air == null) continue;
-                var sum = otherTile2.Air.TotalMoles;
+                var sum = otherTile2.Air.TotalMoles * 0.2f;
```

Comment:
these values should all be cvars so that the old behavior is replicable if people want